src/libhirte/common/cfg.c

```diff
@@ -46,14 +46,12 @@ static uint64_t option_hash(const void *item, uint64_t seed0, uint64_t seed1) {
         return hash;
 }
 
-#pragma GCC diagnostic push
-#pragma GCC diagnostic ignored "-Wunused-parameter"
```

Comment:
I assume that during development there was a variable in `parsing_handler` not being used. This is a warning from gcc when compiling and since we set `werror=true` it will result in an error that fails compilation. The same applies to the linting - this is an overlap of rules - therefore the `NOLINT` was also necessary. 
However, now all variables are used.

Reply:
Ah, right, originally I haven't used the section, but then during development it was added back, but I forgot to remove pragma macros.
Thanks